src/pages/registration/description_tab/projects_field/ProjectContributorRow.tsx

```diff
@@ -26,9 +26,9 @@ import { OrganizationSearchField } from '../../../basic_data/app_admin/Organizat
 import { projectContributorToCristinPerson } from './projectHelpers';
 
 enum ProjectContributorFieldName {
-  Type = 'type',
+  Type = 'roles[0].type',
```

Comment:
Antar det er det brukt `[0]` her som en bevisst quick-fix som vi snakket om, og at vi ikke bryr oss om resten for den gamle formen?

Reply:
I så fall trenger du ikke bry deg så hardt om resten av kommentarene i denne fila heller!

Reply:
Ja, det ble den enkleste løsningen siden vi ikke skal ha dette viewet i mer enn 1-3 uker til. Å bruke denne løsningen øker også sjansen for at vi bare har viewet i 1 uke til og ikke 3.